api/src/pcapi/alembic/versions/20220106T174108_eb6c7fc2d56f_add_indexes_to_bookings.py

```diff
@@ -0,0 +1,47 @@
+"""add indexes to bookings
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = "eb6c7fc2d56f"
+down_revision = "f6efe8c78450"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("COMMIT")
+    op.execute("SET SESSION statement_timeout = '300s'")
+    op.execute(
+        """
+        CREATE INDEX CONCURRENTLY IF NOT EXISTS
+            "ix_booking_is_used"
+        ON
+            booking ("isUsed")
+        WHERE
+            "isUsed" is false
+        """
```

Comment:
**(idée, non bloquant)** J'ai cru comprendre que `isUsed` allait prochainement subir le même sort que `isCancelled` (cf. PR de Cyril) : il sera supprimé, au profit de `status`. Est-ce qu'on ne pourrait pas déjà utiliser `status` au lieu de filtrer sur  `isUsed` ? Autre piste : filtrer sur `"dateUsed" IS NOT NULL` permettrait peut-être d'utiliser l'index existant sur `dateUsed` ?

Reply:
Bien vu, merci. Pour `dateUsed`, ça ne permet pas de savoir si elle a été annulée et donc on devrait quand même regarder la colonne `status`, non ?